models/java/README.md

```diff
@@ -0,0 +1,70 @@
+# gbfs-java-model
+
+![Maven Central Version](https://img.shields.io/maven-central/v/org.mobilitydata.gbfs/gbfs-java-model.svg)
+
+Generates Java model from GBFS json schema using jsonschema2pojo with jackson2 annotations.
+
+In order to avoid naming conflicts in generated classes, each feed gets its own package, and all
+classes are prefixed with "GBFS".
+
+Packages include GBFS versions to accommodate for future releases which may include older versions,
+since some consumers may need to handle multiple non-compatible versions in the same application 
+(i.e. you should be able to parse v2.1 feeds with the v2.2 package, but v3.0 package would not
+be able to parse v2.2 feeds).
+
+## Add the Dependency 
+
+To use the `gbfs-java-model` classes in your own project, you need to add
+an appropriate dependency.  We publish our module to the [Maven Central Repository](http://search.maven.org/)
+so that it can be easily referenced by Java build tools like Maven, Ivy, and Gradle.
+
+For [Maven](http://maven.apache.org/), add the following to your `pom.xml`
+dependencies section:
+
+```xml
+<dependency>
+  <groupId>org.mobilitydata</groupId>
+  <artifactId>gbfs-java-model</artifactId>
+  <version>1.0.0</version>
+</dependency>
+```
+
+For [Gradle](https://www.gradle.org/), add the following to your `build.gradle`
+dependecies section:
+
+```
+implementation group: 'org.mobilitydata', name: 'gbfs-java-model', version: '1.0.0'
+```
+
+Make sure the Maven central repository is referenced by your project.
+
+## Example Code
```

Comment:
@testower would you be able to help with the example code section?

Reply:
```java
// Assume you use Jackson's ObjectMapper for serializing / deserializing,
// this is just one of many options
ObjectMapper objectMapper = new ObjectMapper();

// Assume some InputStream containing a serialized instance of vehicle_status.json v3.0
// Deserialize into corresponding java class
VehicleStatus vehicleStatus = objectMapper.readValue(inputStream, VehicleStatus.class);

// It can be serialized in a similar fashion,
objectMapper.writeValue(outputStream, vehicleStatus);
```

Reply:
This is the simplest possible example I think. Do you think something more advanced?

Reply:
wouldn't `VehiclStatus.class` be `GBFSVehicleStatus.class`? I'm also thinking of adding the import `import org.mobilitydata.gbfs.v3_0.GBFSVehicleStatus;`

Reply:
Indeed you're right, I glossed over the details :)